developer-rchain-coop/src/views/token-swap.hbs

```diff
@@ -16,49 +16,82 @@
                 <div class="title">What is a RHOC?</div>
-                <div class="body">The RHOC is an ERC20 style token created on Ethereum by the RChain cooperative. The contract address is 0x168296bb09e24a88805cb9c33356536b980d3fc5.</div>
+                <div class="body">
+                    <p>The RHOC was an ERC20 token created on Ethereum by the RChain cooperative. The contract address is 0x168296bb09e24a88805cb9c33356536b980d3fc5.</p>
+                    </div>
                 <div class="title">What is a REV?</div>
-                <div class="body">REV is the primary cryptocurrency of the RChain blockchain and is used to bond stake RChain’s CBC Casper proof-of-stake consensus system.</div>
+                    <div class="body">
+                        <p>REV is the primary cryptocurrency of the RChain blockchain and is used to bond stake RChain’s CBC Casper proof-of-stake consensus system.</p>
+                    </div>
+                <div class="title">When was the REV issuance?</div>
+                <div class="body">
+                    <p>
+                    RChain Main net launched on February 25, 2020. REV was issued at that time on the new network.
+                    </p>
+                </div>
+                <div class="title">What does REV issuance mean?</div>
+                <div class="body">
+                    <p>Technically, the event was simply an issuance of the REV utility token, determined by the wallet balances of existing RHOC holders. There is no actual swap. The RHOC will continue to exist on the Ethereum blockchain, even though it is worthless. <b><i>And you will never need to give your RHOC to anyone to receive your REV.</i></b> We designed this process to minimize the risk of phishing. So, if anyone asks you to “send your RHOC” to them, you are being phished.</p>
+                </div>
+                <div class="title">I have RHOC. What do I need to do to receive my REV?</div>
+                <div class="body"> 
+                    <p>If your RHOC is in an etherium account you control, i.e:</p>
+                    <ul>
+                        <li class="body"> You control your private key</li>
+                        <li class="body"> Your RHOC was not in a multisig wallet or smart contract.</li>
+                    </ul>
+                    <p>If both the above are the case then do the following.</</p>
+                </div>
+                <div class="title">How can I see my REV balance?</div>
+                <div class="body">
+                    <p>There will be a few methods.' Here are two:</p>
+                </div>
+                    <ul>
+                        <li class ="body">View with <a class="inline-link" target="_blank" href="https://revdefine.io/#/explorer/rev">Revdefine</a> block explorer</li>
+                        <li class ="body">View with <a class="inline-link" target="_blank" href="https://myrchainwallet.com">MyRchainWallet</a></li>
```

Comment:
linking directly to a wallet is even riskier; it encourages people to put their private keys into that software. I'm inclined to link to a discussion forum such as #rhoc2rev and encourage people to ask about block explorers and wallets there.

We could perhaps use github issues for public forum style discussion of FAQs that are changing. Or stackoverflow or reddit.

Reply:
Disclaimer should offer some protection to Co-op, but doesn't much help users.
#rhoc2rev Discord discussion channel is linked for those looking for help deciding which tools to trust.